kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java

```diff
@@ -292,7 +290,7 @@ private synchronized void doApply(Map<String, Object> networkConfiguration) thro
 
     private synchronized void manageConfiguredInterfaces(List<String> configuredInterfaces,
             NetworkProperties properties) throws DBusException {
-        List<String> availableInterfaces = getAllDeviceInterfaceNames();
+        List<String> availableInterfaces = getInterfaces();
```

Comment:
Does it make sense to call it `getDeviceIds`? Indeed, under the hood it calls `getDeviceId` for all the available and supported devices.